internal/rukpak/convert/registryv1.go

```diff
@@ -107,6 +107,7 @@ func RegistryV1ToHelmChart(ctx context.Context, rv1 fs.FS, installNamespace stri
 	}
 
 	chrt := &chart.Chart{Metadata: &chart.Metadata{}}
+	chrt.Metadata.Annotations = reg.CSV.GetAnnotations()
```

Comment:
I've already approved, but didn't think about this until after - are there any tests that can/should be updated to verify that this is working as expected?

Reply:
Yeah, we should:
1. Make sure our test CSV has some annotations
2. Make sure (via test assertions) that those annotations make it into chart metadata annotations.